src/IHaskell/Eval/Evaluate.hs

```diff
@@ -229,13 +229,26 @@ data EvalOut = EvalOut {
     evalComms :: [CommInfo]
   }
 
+cleanString :: String -> String
+cleanString x =  if allBrackets then clean else str
+  where
+    str = strip x
+    l = lines str
+    allBrackets = all (fAny [isPrefixOf ">", null]) l
+    fAny fs x = any ($x) fs
+    clean = unlines $ map removeBracket l
+    removeBracket (x:xs) = xs
```

Comment:
This should be 

```haskell
removeBracket ('>':xs) = xs
```
instead of

```haskell
removeBracket (x:xs) = xs
```
Otherwise the `other` case is never reached.